patches/third_party-blink-renderer-modules-service_worker-service_worker_content_settings_proxy.h.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/third_party/blink/renderer/modules/service_worker/service_worker_content_settings_proxy.h b/third_party/blink/renderer/modules/service_worker/service_worker_content_settings_proxy.h
+index a6405c3a53a56e8563610380f80ccda0e1689757..955543d1d55bc542b71c3633da7f5e0b9d8af57d 100644
+--- a/third_party/blink/renderer/modules/service_worker/service_worker_content_settings_proxy.h
++++ b/third_party/blink/renderer/modules/service_worker/service_worker_content_settings_proxy.h
+@@ -38,6 +38,7 @@ class ServiceWorkerContentSettingsProxy final
+   // Blocks until the response arrives.
+   bool AllowStorageAccessSync(StorageType storage_type) override;
+ 
++  BRAVE_SERVICE_WORKER_CONTENT_SETTINGS_PROXY_H
```

Comment:
can be fixed the same way as in `shared_worker_content_settings_proxy.h`

Reply:
Fixed in https://github.com/brave/brave-core/pull/11452/commits/2a456ab2da3dfe5f05a73b9e3bc7c07872c2c87d